source/index.rst

```diff
@@ -26,29 +28,14 @@ Community translations can be found in a variety of languages in the bottom-left
 
    ^^^
 
-   If you are a returning team, please check out the overview of changes from 2021 to 2022 and the known issues.
+   If you are a returning team, click the button below to view a list of articles to refresh on.
 
    +++
 
-   .. div:: container-fluid p-0
-
-      .. div:: row
-
-         .. div:: col-sm pl-1 pr-1
-
-            .. link-button:: /docs/yearly-overview/yearly-changelog
-               :type: ref
-               :text: Changelog
-               :classes: btn-outline-primary btn-block
-
-      .. div:: row
-
-         .. div:: col-sm pl-1 pr-1 pt-1
-
-            .. link-button:: /docs/yearly-overview/known-issues
-               :type: ref
-               :text: Known Issues
-               :classes: btn-outline-primary btn-block
+   .. link-button:: /stubs/returning-teams-stub
+      :type: ref
+      :text: Popular Articles
```

Comment:
I don't like the name popular articles as it implies good things, not necessary things